### ISSUE TITLE
plan for TGN auto-suggest 

### ISSUE THREAD

**Issue by melissaagill** [gri-is/lodjob#33]

**Comment by nlomeli** [comment]
Sent the following message to Gregg Garcia to inquire about TGN data sourcing:

Hi Gregg,
 
We’re working on the data/Elasticsearch schema for a TGN/place reconciliation service.  While the linked data itself is hard to parse visually to determine what’s relevant to include (e.g. http://vocab.getty.edu/tgn/6006299?inference=all ), I noticed that the “Semantic View” that you generate is very helpful:
 
http://www.getty.edu/vow/TGNFullDisplay?find=&place=&nation=&subjectid=6006299&english=Y
 
How are you generating these semantic views on the back end? You seem to be doing some dereferencing there that we should avoid duplicating if possible.  In the language of the semantic view, these are the fields that I’m thinking will be most relevant for reconciliation:
 
Record type
Name (preferred)
Names (nonpreferred) (and languages)
Coordinates (lat and long)
Hierarchical position (one or more levels up)
Place type (preferred)
Place types (nonpreferred)
Related geographic places

**Comment by nlomeli** [comment]
These are the steps needed to launch this service:

1. acquire or synthesize appropriately schematized place records from vocabs dump 
2. repurpose ULAN scripts for loading TGN data into Elasticsearch
3. clone and repurpose or simply add new endpoints to existing Flask app which services ULAN reconciliation
4. repurpose test scripts to analyze reconciliation performance

**Comment by joshuago78** [comment]
@azaroth42 would you mind weighing in on this?

**Comment by azaroth42** [comment]
What can I do to help? :)

**Comment by joshuago78** [comment]
@azaroth42 I think there is some uncertainty as to what the documents in ES should look like (i.e. what is the extent of de-referencing?). Is Nathan's list of 8 fields above sufficient?
